src/Serializer/BaseSerializerHandler.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Form\Serializer;
+
+use JMS\Serializer\Context;
+use JMS\Serializer\GraphNavigator;
+use JMS\Serializer\VisitorInterface;
+use Sonata\Doctrine\Model\ManagerInterface;
+
+/**
+ * @author Sylvain Deloux <sylvain.deloux@ekino.com>
+ */
+abstract class BaseSerializerHandler implements SerializerHandlerInterface
+{
+    /**
+     * @var ManagerInterface
+     */
+    protected $manager;
+
+    /**
+     * @var string[]
+     */
+    protected static $formats;
```

Comment:
```suggestion
    protected static $formats = [];
```

Reply:
This is set by bundle config. I think this is good suggestion, becouse someone can use it without bundle load.